nntrainer/layers/pooling2d_layer.cpp

```diff
@@ -125,6 +143,13 @@ void Pooling2DLayer::forwarding(RunLayerContext &context, bool training) {
 }
 
 void Pooling2DLayer::calcDerivative(RunLayerContext &context) {
+  std::vector<props::PoolSize> pool_size =
+    std::get<std::vector<props::PoolSize>>(pooling2d_props);
+  std::array<props::Stride, POOLING2D_DIM> stride =
```

Comment:
Same here, you can use `auto &stride` instead :)

Reply:
Done